add task solution

SergiyVS opened the pull request:
https://sergiyvs.github.io/layout_calendar/

SergiyVS committed:
add task solution

loralevitska commented on src/styles/blocks/calendar.scss:
```diff
@@ -0,0 +1,70 @@
+  width: $calendar-width;
+  padding: $calendar-paddings;
+
+  background-color: #080;
```
It is a good practice to make all colors variable

loralevitska commented on src/styles/blocks/calendar.scss:
```diff
@@ -0,0 +1,70 @@
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
```
Move it to `_variable.scss`

loralevitska reviewed (changes requested):
Add demo_link to task description for review 
![image](https://github.com/mate-academy/layout_calendar/assets/70009205/263a1805-5a66-469d-b36d-fe917eab590a)

SergiyVS committed:
Changing by comments
1. colors made variable
2. Array $start-days transferred to _variables.scss

SergiyVS commented:
https://sergiyvs.github.io/layout_calendar/

polosanya commented on src/styles/blocks/calendar.scss:
```diff
@@ -0,0 +1,61 @@
+    font-size: 30px;
+    background-color: $day-color;
+
+    transition: transform 0.5s;
```
It would be great to create a variable for transition duration, so you can change it later if needed

polosanya commented on src/index.html:
```diff
@@ -1,12 +1,44 @@
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="calendar calendar--start-day--thu calendar--month-length--31 page__calendar">
```
Do you really need a page__calendar class? You don't use it

polosanya reviewed (approved):
Good job!

SergiyVS committed:
changed
1. created a variable for transition duration
2. page__calendar class deleted. Not in use
